api/alembic/versions/b557469a7727_modify_unique_constraint_on_planning_.py

```diff
@@ -0,0 +1,30 @@
+"""Modify unique constraint on planning_weather_stations
+
+Revision ID: b557469a7727
+Revises: 1caf3488a340
+Create Date: 2022-01-17 10:15:33.448252
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'b557469a7727'
+down_revision = '1caf3488a340'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint('planning_weather_stations_station_code_key', 'planning_weather_stations', type_='unique')
+    op.create_unique_constraint(None, 'planning_weather_stations', ['station_code', 'planning_area_id'])
```

Comment:
Can we replace `None` with an explicit name from the constraint? I think the library auto-creates a name but it broke when I was writing a migration before and it's not that descriptive. The name can be a variable used in both up/down functions.